profiles/lib/directional.lua

```diff
@@ -56,4 +56,37 @@ function Directional.get_values_by_set(way,data,keys)
   return forward, backward
 end
 
+-- look through a sequence of keys combined with a prefix
+-- e.g. for the sequence [motorcar,motor_vehicle,vehicle] and the prefix 'oneway' search for:
+-- oneway:motorcar
+-- oneway:motor_vehicle
+-- oneway:vehicle
+
+function Directional.get_value_by_prefixed_sequence(way,seq,prefix)
```

Comment:
these helpers are not directly related to directional (forward/backward) tags. either they could be moved to a separate file, or we could rename the Directional helper to something more general.